src/components/ChangelogDialog/components/Picture/Picture.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import {block} from '../../../utils/cn';
+import {Loader} from '../../../Loader';
+
+import './Picture.scss';
+
+const b = block('changelog-dialog-picture');
+
+type LoadingState = 'loading' | 'loaded' | 'error';
+
+const SHOW_LOADER_TIMEOUT = 150;
+
+export interface PictureProps {
+    className?: string;
+    src: string;
+    alt?: string;
+    ratio?: number;
+}
+
+export function Picture({className, src, alt, ratio}: PictureProps) {
```

Comment:
There should be at least emty string in `alt`

```suggestion
export function Picture({className, src, alt = '', ratio}: PictureProps) {
```

Reply:
Why? `alt` is optional parameter, if it is not passed then the `alt` attribute will not be rendered.

Reply:
it is required for screen readers

Reply:
Will a screen reader read differently if there is no alt attribute and alt=""?

Reply:
it would be considered as decorative image

Reply:
And, it is a decorative image. If consumers of the component need `alt` support, they can specify it in the data.

Reply:
alt could be undefined, and in this case there would be no `alt` attribute and this is against WCAG recommendations